pkgs/development/interpreters/wasmtime/default.nix

```diff
@@ -1,27 +1,29 @@
-{ rustPlatform, fetchFromGitHub, lib, stdenv }:
+{ rustPlatform, fetchFromGitHub, Security, lib, stdenv }:
 
 rustPlatform.buildRustPackage rec {
   pname = "wasmtime";
-  version = "8.0.1";
+  version = "9.0.0";
 
   src = fetchFromGitHub {
     owner = "bytecodealliance";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-xSHwR2MGL49VDKjzAh+xYHbLz3FFg3KYVBjALVgKSQI=";
+    hash = "sha256-9ga7BKJoaw7naX8t4o+zNnWkjIvSII5oVRM0dYMrseo=";
     fetchSubmodules = true;
   };
 
-  cargoHash = "sha256-A2JhjRFKPltHubiJYHBXj2H4cdU43Y2x6UjEpRGPX7U=";
+  cargoHash = "sha256-GkL78aAIGdSlcxeRTIVp1jcXIg1ZtvB2LNIoPEViNcs=";
 
   cargoBuildFlags = [ "--package" "wasmtime-cli" "--package" "wasmtime-c-api" ];
 
   outputs = [ "out" "dev" ];
 
+  propagatedBuildInputs = lib.optionals stdenv.isDarwin [ Security ];
```

Comment:
Why propagatedBuildInputs as opposed to buildInputs?

Reply:
Not good reason here, sorry for the noise :/